### ISSUE TITLE
SKU is Optional in Shopify, so it Can be Nullable in ProductVariantCheckout

### ISSUE THREAD

**Issue by asadamatic** [imsujan276/shopify_flutter#43]
When I try to add a product variant that does not have an `sku`  it throws the following error.

**Error**
```
Unhandled Exception: type 'Null' is not a subtype of type 'String' in type cast
```

**Stack Trace**
```
#0      _$$ProductVariantCheckoutImplFromJson (package:shopify_flutter/models/src/checkout/product_variant_checkout/product_variant_checkout.g.dart:15:24)
#1      new _$ProductVariantCheckoutImpl.fromJson (package:shopify_flutter/models/src/checkout/product_variant_checkout/product_variant_checkout.freezed.dart:340:7)
#2      _$ProductVariantCheckoutFromJson (package:shopify_flutter/models/src/checkout/product_variant_checkout/product_variant_checkout.freezed.dart:19:34)
#3      new ProductVariantCheckout.fromJson (package:shopify_flutter/models/src/checkout/product_variant_checkout/product_variant_checkout.dart:31:7)
#4      LineItem.fromGraphJson (package:shopify_flutter/models/src/checkout/line_item/line_item.dart:29:36)
#5      JsonHelper.lineItems.<anonymous closure> (package:shopify_flutter/models/json_helper.dart:19:30)
#6      MappedListIterable.elementAt (dart:_i<…>
```

It points to `ProductVariantCheckoutImplFromJson` where SKU is expected to be available.

**Comment by asadamatic** [comment]
Once I get this error, adding any item to the same checkout does not work, and I have to create another checkout.

**Comment by asadamatic** [comment]
@imsujan276 The SKU is now nullable is the `ProductVariant`, but it's still required `ProductVariantCheckout`.

**Comment by imsujan276** [comment]
Thank you for bringing it out. 

It has been fixed in v1.5.8

**Comment by asadamatic** [comment]
Thanks a lot @imsujan276
